DEVELOPERS.md

```diff
@@ -523,3 +526,61 @@ inversion of control so the calling view can decide the context in which
 presenters are used.
 This will most likely affect the template used for each event, and where each
 object links to, if anywhere.
+
+## Interfaces
+
+Descriptions of interfaces between this repo or container and others. These
+interfaces can be changed, through coordination with the relevant teams.
+
+Except where mentioned otherwise, URLs are relative to the root of the
+job-server API endpoint (https://jobs.opensafely.org/api/v2 in production).
+
+### [job-runner]
+
+[job-runner] is the container that runs in a secure backend and executes
+[JobRequest]s initiated by users of job-server.
+
+This interacts with [jobserver/api/jobs.py] It uses the `JobRequestAPIList`
+endpoint (`GET /job-requests/`) for reading `JobRequests`. It uses the
+`JobAPIUpdate` endpoint (`POST /jobs/`) for updating the `Jobs` table.
+(Current as of 2024-09.)
+
+ Refer to the documentation of [jobrunner.sync] for job-runner's documentation
+ of this interface.
+
+[job-runner]: https://github.com/opensafely-core/job-runner
+[jobrunner.sync]: https://github.com/opensafely-core/job-runner/blob/main/DEVELOPERS.md#jobrunnersync
+[JobRequest]: jobserver/models/job_request.py
+[jobserver/api/jobs.py]: jobserver/api/jobs.py
+
+### [airlock]
+
+[airlock] is the container that [OutputCheckers] interact with to check and
```

Comment:
Nitpick: I'd suggest one of:
* "users with the `OutputChecker` role"
* "users with the output checker role"
* "output checkers"

My intention would be to clarify whether we're referring to something within the narrow job-server domain or something within the wider Bennett Institute domain.

Reply:
Good point, thanks, gone with the first.